### PR TITLE
Add article about Tor filtering

### PR DESCRIPTION
**Pull request by ss-orlova**

### CODE REVIEW

File: docs/adguard-for-mac/solving-problems/tor-filtering.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Setting up filtering in Tor Browser
+sidebar_position: 10
+---
+
+By default, Tor Browser is added to AdGuard’s list of filtered applications. However, to filter it, AdGuard needs to change Tor's security settings. This will make Tor trust AdGuard's CA certificate.
+
+If you decide to to filter Tor and check its box in *Preferences* → *Network* → *Applications*, AdGuard will show you this dialog window:
```
Comment (vozersky) [inline docs/adguard-for-mac/solving-problems/tor-filtering.md:8]: "to to"

File: docs/adguard-for-mac/solving-problems/tor-filtering.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+[AdGuard dialog window](https://cdn.adtidy.org/content/kb/ad_blocker/mac/tor-setup.png)
+
+By updating Tor setting, AdGuard will set the following:
```
Comment (vozersky) [inline docs/adguard-for-mac/solving-problems/tor-filtering.md:12]: settings

**Review by vozersky — APPROVED** [review]

File: docs/adguard-for-mac/solving-problems/tor-filtering.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Setting up filtering in Tor Browser
+sidebar_position: 10
+---
+
+By default, Tor Browser is added to AdGuard’s list of filtered applications. However, to filter it, AdGuard needs to change Tor's security settings. This will make Tor trust AdGuard's CA certificate.
+
+If you decide to filter Tor and check its box in *Preferences* → *Network* → *Applications*, AdGuard will show you this dialog window:
```
Comment (el-termikael) [inline docs/adguard-for-mac/solving-problems/tor-filtering.md:8]: dialog box

File: docs/adguard-for-mac/solving-problems/tor-filtering.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+[AdGuard dialog window](https://cdn.adtidy.org/content/kb/ad_blocker/mac/tor-setup.png)
```
Comment (el-termikael) [inline docs/adguard-for-mac/solving-problems/tor-filtering.md:10]: и тут

File: docs/adguard-for-mac/solving-problems/tor-filtering.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+title: Setting up filtering in Tor Browser
+sidebar_position: 10
+---
+
+By default, Tor Browser is added to AdGuard’s list of filtered applications. However, to filter it, AdGuard needs to change Tor's security settings. This will make Tor trust AdGuard's CA certificate.
```
Comment (el-termikael) [inline docs/adguard-for-mac/solving-problems/tor-filtering.md:6]: I guess, it's Tor or the Tor Browser

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Pull request merged**